queries.sql

```diff
@@ -7,3 +7,105 @@ SELECT name,escape_attempts from animals WHERE weight_Kg > 10.5;
 SELECT * from animals WHERE neutered=true;
 SELECT * from animals WHERE NOT name='Gabumon';
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+/* Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made.
+   Then roll back the change and verify that species columns went back to the state before tranasction. */
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
```

Comment:
@addod19 I thought I had addressed the required changes. Kindly elaborate more on what you exactly need in this requirement

Reply:
@Tumworobere, please contact me on Slack.  I can help.

Reply:
Hey @Tumworobere, I'll try to explain to the best of my ability. 💪 

After each change to the table AND after each rollback, you need to use a SELECT statement to verify that the change was actually made.

The goal is for you to observe and understand how these changes, rollbacks, and commits behave.


Example: 

```SQL
BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made THESE ARE THE STATEMENTS THAT YOU ARE MISSING.
```

Please, use a SELECT statement to verify the data after each UPDATE, ROLLBACK, COMMIT and DELETE, and your project should be approved!!  You can do it!!!

Reply:
Hey, @Meltrust I worked on the required changes you requested. I used this statement to verify data"TABLE animals"  instead of the select statement which I thought was also okay. But I have used the select statement as requested.Thanks for your time.